keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -687,20 +997,92 @@ We will extend this approach:
 Note that this does allow `ValidatingAdmissionPolicy` to match
 `ValidatingWebhookConfiguration`.
 
-We are also considering not allowing a `ValidatingAdmissionPolicy` to match the
-configuration CRD kind that is uses for config. It would be possible to recover
-from this since the `ValidatingAdmissionPolicy` would remain accessible, but
-might require a privileged user do the repair.
-
-TODO: We should consider loosening this up and allow admission configuration to
-intercept/guard writes to admission configuration while preventing deadlock -
-Add feature to configure a set of webhooks to intercept other webhooks
-https://github.com/kubernetes/kubernetes/issues/101794.
+Note: In the future we may further loosen this up and allow admission
+configuration to intercept/guard writes to admission configuration while
+preventing deadlock - Add feature to configure a set of webhooks to intercept
+other webhooks https://github.com/kubernetes/kubernetes/issues/101794.
 
 Alternative considered: Each `ValidatingAdmissionPolicy` has a "level", a
 `ValidatingAdmissionPolicy` can match another `ValidatingAdmissionPolicy` of a
 higher level. This could be added later.
 
+#### Reporting violations
```

Comment:
How does violation reporting interact with policy frameworks? For example, Gatekeeper also reports the # of webhook violations as a prometheus metric. If the goal is to have higher-level frameworks provide a single pane of glass (e.g. they are also polyfilling the stuff that embedded policy wont/can't do), then some kind of two-way communication is needed.

If we are okay with fragmented reporting, then we probably should restate how this KEP is positioning itself WRT extensibility.

Reply:
This should be titled "Reporting violations back in response to client". We will need metrics.

Reply:
Clarified and added a separate metrics section.